Add support `SubFolder::class` middleware. (#269)
yiisoft/app: README.md

## Configuration

You can find configuration in [config](/config/) directory. There are multiple
configs, and the most interesting is [params.php](/config/params.php). Below there are details about its sections:

## Using SubFolder Middleware for URL Routing

> If you want to use subfolder middleware for URL routing, you need to change the following code to the `config/params.php` file:

Reviewer: ```suggestion
If you want to use subfolder middleware for URL routing, you need to adjust `config/params.php` file.
```

> For example directory develop in the root of the project, and public directory is in the `yii3/public` folder.

Reviewer: ```suggestion
For our example let's assume that web server root is pointing to the all projects root. There is `yii3` project with its `yii3/public` directory that should be accessed as `http://localhost/yii3/public`.

> Note: While being a common practice for local development, it is recommended to prefer separate hosts for separate projects pointint directly to `public` directory.

Here's how `config/params.php` should be adjusted:
```

```php
'app' => [
    'prefix' => '/yii3/public',
],
```

> run the following command:

Reviewer: ```suggestion
To test it in action run the following command:
```

```bash

> /d/develop$>php -S 127.0.0.1:8080 //web server entry point

Reviewer: ```suggestion
php -S 127.0.0.1:8080 <all projects root path>
```

```

> using in browser `http://localhost:8080/yii3/public` to access the application.

Reviewer: ```suggestion
Now you can use `http://localhost:8080/yii3/public` to access the application.
```
